src/main/java/core/basesyntax/Application.java

```diff
@@ -2,6 +2,10 @@
 
 public class Application {
     public static void main(String[] args) {
-        // create three balls using class Lottery and print information about them in console
+        Lottery lottery1 = new Lottery();
```

Comment:
```suggestion
        Lottery lottery = new Lottery();
```

It's bad practice to use numbers in variable names